new version

thiagolif opened the pull request:
Hi! I have made a new version of kisscommunity.org, here is a live view: https://thiagolif.github.io/kisscommunity.org/
hope you like it

thiagoferreiradev committed:
new version

thiagoferreiradev committed:
minor fix for use in github pages

jedahan commented on index.html:
```diff
@@ -1,30 +1,54 @@
+        <link href="./assets/css/normalize.css" rel="stylesheet">
+        <link href="./assets/css/skeleton.css" rel="stylesheet">
+        <link href="./assets/css/style.css" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap" rel="stylesheet">
```
Could you download Open Sans and include it directly instead of injecting a stylesheet? Should be much smaller that way.

thiagolif replied:
done

jedahan commented on index.html:
```diff
@@ -1,30 +1,54 @@
+            <div class="container">
+                <h1>KISS Linux Community</h1>
+                <blockquote>when dylan says "no", we say "maybe"</blockquote>
+                <blockquote><a href="https://kisslinux.org/" target="_blank">KISS Linux</a> is an independent Linux(R) distribution with a focus on simplicity and the concept of less is more.
```
Please remove `target="_blank"` in all `<a` tags - it changes browser defaults, and is nicer to be up to the user.

thiagolif replied:
done

jedahan commented on old/index.html:
```diff
@@ -0,0 +1,30 @@
+<head>
```
Olease remove all unused files (the entire `old/` directory can go)

thiagolif replied:
done

jedahan reviewed (approved):
Thanks for the update. Its a bit heavier of a site but nice re-organization of sections.

jedahan commented:
I commented on three suggestions - if you'd like to take them on let me know, otherwise I can implement them if there are no objections.

jedahan commented:
One more note: the original site supports a light-mode and dark-mode, which was a user request. Is there a way to do that with this update?

thiagolif commented:
> I commented on three suggestions - if you'd like to take them on let me know, otherwise I can implement them if there are no objections.

im doing the changes you asked right now, soon i'll commit them

thiagoferreiradev committed:
added dark mode
open sans font now is directly included
removed normalize.css and bloated content of skeleton.css

thiagolif commented:
> One more note: the original site supports a light-mode and dark-mode, which was a user request. Is there a way to do that with this update?

done and thanks for the tips

jedahan commented:
Thanks for the quick updates
